sdk/packages/core/src/api/validateOrder.ts

```diff
@@ -83,6 +86,16 @@ export async function validateOrder(apiBaseUrl: string, encodedOrder: string) {
   return json as ValidationResponse
 }
 
+export async function validateOrder<abis extends OptionalAbis>(
+  apiBaseUrl: string,
+  order: Order<abis>,
+) {
+  return await validateOrderWithEncoded(
+    apiBaseUrl,
+    encodeOrderForValidation(order),
```

Comment:
why not just `encodeOrder`? matched `encodeQuote` pattern. do we have a different variation of order encoding

Reply:
`encodeOrder` was already used to encode the order data for the contract, I renamed it to `encodeOrderData` and `encodeOrderForValidation` to `encodeOrderRequest` so hopefully it's more explicit?